packages/back-end/src/services/experiments.ts

```diff
@@ -1663,6 +1663,20 @@ export function putMetricApiPayloadToMetricInterface(
         metric.windowSettings.windowValue =
           behavior.conversionWindowEnd - behavior.conversionWindowStart;
       }
+    } else {
```

Comment:
Because this is designed to work with only partial behavior updates, I think this check should happen first (e.g. `if (typeof behavior.windowSettings !== "undefined")` then do this code, then `else if (typeof behavior.conversionWindowStart !== "undefined")`.

Otherwise right now if you don't specify `windowSettings` but make other behavior updates, you'll just overwrite the window with the defaults.

Switching the order also helps us prefer the non-deprecated field.